addons/account/tests/test_account_move_reconcile.py

```diff
@@ -194,6 +194,19 @@ def assertAmountsGroupByAccount(self, amount_per_account):
                 "Amount currency of %s is incorrect" % account.name,
             )
 
+    def create_move_payment(self, move, payment_amount, with_outstanding_account=False):
+        payment = self.env['account.payment.register'].with_context(
+            active_model='account.move',
+            active_ids=move.ids,
+        ).create({
+            'amount': payment_amount,
+            'payment_method_line_id':
+                self.company_data['default_journal_bank'].inbound_payment_method_line_ids.filtered_domain([
+                    ('payment_account_id', '!=' if with_outstanding_account else "=", False),
+                ])[0].id,
+        })._create_payments()
+        return payment
```

Comment:
For the next reviewer: 
- We transform the create_move_payment as an helper for the others tests, (removed the assertEqual since for me it's weird to have that in a create function